packages/atomic/storybookUtils/search/recs-interface-wrapper.tsx

```diff
@@ -31,6 +35,9 @@ export const wrapInRecommendationInterface = (
         ...config,
       });
     });
+    if (skipFirstQuery) {
+      return;
+    }
     await step('Execute the first search', async () => {
       await searchInterface!.getRecommendations();
```

Comment:
I think in general you don't need the non-null assertion on the `searchInterface` variable because `canvas.findByTestId<HTMLAtomicRecsInterfaceElement>` [somehow] always returns a Promise that resolves to an `HTMLAtomicRecsInterfaceElement`.

Reply:
Thanks !!!!